columnphysics/icepack_kinds.F90

```diff
@@ -17,10 +18,18 @@ module icepack_kinds
                             char_len_long  = 256, &
                             log_kind  = kind(.true.), &
                             int_kind  = selected_int_kind(6), &
+#if defined (NO_I8)
```

Comment:
Minor point, but why `#if defined` and not `#ifdef` ? I think they do the same thing but maybe we should stick to one or the other?

Reply:
In CICE, there is a mixture of #ifdef and #if defined.  The NO_I8 in CICE has #if defined, so I made the icepack format match that.  I could change it here, but then would probably want to change it in CICE as well.  If we want to take this step, I'm happy to do it.  I'm personally not that concerned about mixing the styles, although agree a single approach is nicer (but then requires diligence to maintain it).